sdk/src/main/kotlin/org/onflow/flow/sdk/AsyncFlowAccessApi.kt

```diff
@@ -64,6 +66,10 @@ interface AsyncFlowAccessApi {
 
     fun getLatestProtocolStateSnapshot(): CompletableFuture<FlowAccessApi.AccessApiCallResponse<FlowSnapshot>>
 
+    fun getProtocolStateSnapshotByBlockId(blockId: FlowId): CompletableFuture<FlowAccessApi.AccessApiCallResponse<FlowSnapshot>>
+
+    fun getProtocolStateSnapshotByHeight(height: Long): CompletableFuture<FlowAccessApi.AccessApiCallResponse<FlowSnapshot>>
+
```

Comment:
@lealobanov would be good to address the doc suggestions here and perhaps more broadly across the codebase since so much new stuff has been add. Fine if you want to do that in a new PR also.

Reply:
@lealobanov - the first doc suggestions seems reasonable, could apply that across new entities where there's not much doc yet. Covering that in a new PR is fine too. The rest i don't think are needed